.setup/submitty_crontab

```diff
@@ -16,3 +16,5 @@
 0 2 * * * submitty_daemon python3 /usr/local/submitty/sbin/auto_rainbow_scheduler.py
 # Update OS Info and Service Health every hour
 0 * * * * submitty_daemon   /usr/local/submitty/sbin/update_worker_sysinfo.sh UpdateSystemInfo
+# Run the repair_services.sh script every minute
+* * * * * submitty_daemon   sudo /usr/local/submitty/sbin/repair_services.sh
```

Comment:
Let's back this off to once an hour.   We don't need to check it quite so frequently.
We should update the worker health message to once an hour as well.  

I don't want to burden the server with things we expect to (essentially) never need.